src/Symfony/Component/RateLimiter/Policy/FixedWindowLimiter.php

```diff
@@ -29,19 +29,8 @@ final class FixedWindowLimiter implements LimiterInterface
 {
     use ResetLimiterTrait;
 
-    private $id;
-    private $limit;
-    private $storage;
-
-    /**
-     * @var int seconds
-     */
-    private $interval;
-
-    /**
-     * @var LockInterface
-     */
-    private $lock;
```

Comment:
Did you mean to use promotion for these removed props?

Reply:
No, they're declared inside the trait already. I'm backporting this, I guess.

Reply:
#42144